easy-ui-react/src/Button/Button.tsx

```diff
@@ -59,11 +57,6 @@ export const Button = forwardRef<null, ButtonProps>((props, inRef) => {
   } = props;
 
   const ref = useRef(null);
```

Comment:
**suggestion:** you can let me know if i'm missing something, but i believe you can remove this `ref` since it's not being used for anything local to the component